CONTRIBUTING.md

```diff
@@ -89,150 +89,176 @@ effect on highlighting. We will work on improving highlighting in the near futur
 
 #### Misc
 
-```
-@comment
-@debug
-@error for error `ERROR` nodes.
-@none to disable completely the highlight
-@preproc
-@punctuation.delimiter for `;` `.` `,`
-@punctuation.bracket for `()` or `{}`
-@punctuation.special for symbols with special meaning like `{}` in string interpolation.
+```scheme
+@comment  ; line and block comments
+@error    ; syntax/parser errors
+@none     ; completely disable the highlight
+@preproc  ; various preprocessor directives & shebangs
+@define   ; preprocessor definition directives [NOTE: unused]
```

Comment:
Why did you add it if it's not used? (Honestly curious.)

Reply:
I wanted to document every capture that had a `TS*` equivalent.
I didn't add `@macro` and `@structure` since they didn't have one.

Reply:
Ah, that makes sense. But we don't look back; `TS*` is dead and gone now ;)